Assets/MRTK/Tests/PlayModeTests/BoundingBoxTests.cs

```diff
@@ -60,7 +61,37 @@ private BoundingBox InstantiateSceneAndDefaultBbox()
             });
 
             bbox.Target = cube;
-            bbox.transform.localScale = boundingBoxStartScale;
+            bbox.Active = true;
+
+            return bbox;
+        }
+
+        /// <summary>
+        /// Instantiates a bounding box at 0, 0, -1.5f
+        /// box is at scale .5, .5, .5
+        /// Target is set to its child
+        /// </summary>
+        private BoundingBox InstantiateSceneAndDefaultBboxWithChildTarget()
+        {
+            
+            var bboxGameObject = new GameObject();
+            bboxGameObject.transform.position = boundingBoxStartCenter;
+            bboxGameObject.transform.localScale = boundingBoxStartScale;
+            BoundingBox bbox = bboxGameObject.AddComponent<BoundingBox>();
+            var cube = GameObject.CreatePrimitive(PrimitiveType.Cube);
+            cube.transform.parent = bboxGameObject.transform;
+            cube.transform.localScale = Vector3.one;
+            cube.transform.localPosition = Vector3.zero;
+            cube.transform.localRotation = Quaternion.identity;
+
+            MixedRealityPlayspace.PerformTransformation(
+            p =>
+            {
+                p.position = Vector3.zero;
+                p.LookAt(bboxGameObject.transform.position);
+            });
+
+            bbox.Target = cube;
             bbox.Active = true;
```

Comment:
Maybe , InstantiateSceneAndDefaultBbox() signature could be changed to accept a optional 'target' game object, as calling it here could replace some of the duplicated code.

Reply:
Agreed and changed!

Reply:
i'm wondering - why are we not passing the target as a game object as vanessa recommended? there's only one test using this, so the gameobject could just be created there and passed as a target param into this method. Do we want this to be in the default instantiateSceneAndDefaultBBox method? (On the other hand we're deprecating boundingbox, so i don't care too much about this change, if there's a reason to do it that way :) )

Reply:
Changed to taking a game object as parameter now.